### DISCUSSION TITLE
7.1 release

### DISCUSSION THREAD

**Discussion by jcox10** [nginx/ansible-role-nginx-config#372] [General] [answered]
When will 7.1 be released?

**Comment by alessfg** [comment, marked as answer]
There really isn't any cadence to releases (part of the never ending backlog is to figure out one 🙃 ). I usually try to create a new release when there's been relevant changes to the role in the past couple months, which actually means it's probably time to create a new release! I'll hopefully get to this early next week 😄

**Reply by jcox10** [reply]
Thanks!

**Comment by alessfg** [comment]
It's out!

**Comment by jcox10** [comment]
Thank you!
